docs/glossary-and-resources.md

```diff
@@ -23,23 +23,44 @@
+- **formatter** - in MessageFormat, a function that takes a set of parameters (eg: ```code: 'USD', amount: 3.9```), and returns formatted text that can be incorporated into the output string (eg: ```"<span class="cur"><span class="cur-symbol">$</span><span class="cur-amt">3.90</span> <span class="cur-code">USD</span>"```)
+- **function** - in MessageFormat, functionality that is callable in the context of a MessageFormat message to perform a variety of functions, from determining the plural form needed to represent a number, to formatting an option for output.
+- **gender** - In many languages, nouns are considered to have a gender which requires changes to the text around the noun. Common genders in linguistics are feminine, masculine, neuter, common (gendered) or epicene.
+- **grammeme** - a value of a grammatical category, such as "singular" or "plural" for numerical grammer, or "male" or "female" for gendered nouns
+- **implementation** - code written to achieve the intended behavior of output for a given input. It is expected that MessageFormat will have implementations in all popular programming languages.
+- **inflection** - a variation in a word or phrase common in many languages, that expresses further context such as tense, mood, formality, etc.
 - **interchange format** - syntax/file format used to convert the inputs of message formatting into inputs for other systems (ex: l10n systems).
 - **interpolate** - inserting the contents of one string in the middle of another, at places indicated by a pattern / placeholder. See _translation merging_.
+- **LDML** - Locale Data Markup Language - the spec schema that defines CLDR
 - **locale fallback** - offering a reasonable substitute locale when the requested locale's resources are not available. Results may vary depending on context (ex: audio vs. text vs. video).
 - **locale matching** - computing the locale fallback.
+- **markup** - characters present in the MessageFormat source strings that are intended to provide structure, functionality, or formatting to the output string, as opposed to display text.
+- **match** - a mechanism in MessageFormat that allows the message author to specify a variety of outcomes based on the combined values of one or more inputs.
+- **metadata** - Addition information about an MF2 message that aids in translation, or other pre-runtime processes, but has no impact on how the message will be rendered. This may be context notes, screenshots of the string in situ, etc. Metadata should never be relied upon at runtime.
+- **open/close** - syntax within the source test that changes the execution context of the parser from output mode to code mode, and back again.
+- **operand** - A character in a message that denotes an action taken during runtime execution, such as value assignment or comparison.
+- **options** - values passed in, along with an MF2 message, that will be used to determine the output. Examples may be item count in a search results string.
```

Comment:
Using "options" for this is misleading; that's used often for the named arguments of expressions and markup.

For this meaning, I think we usually say "arguments" or "parameters".

Reply:
ok, thanks.  I was inferring from how I'd heard it used in conversation, but I don't think we are all using is consistently yet.